packages/common/src/core/types/IHubProject.ts

```diff
@@ -26,3 +27,16 @@ export enum PROJECT_STATUSES {
   onHold = "onHold",
   complete = "complete",
 }
+
+/**
+ * This type redefines the IHubProject interface in such a way
+ * that it can be consumed by the entity editor.
+ */
+export type IHubProjectEditor = Omit<IHubProject, "extent"> & {
+  extent: IExtent | number[][];
```

Comment:
I don't _think_ extent needs to be here anymore?

Reply:
I remove it, and verified in the harness...